pkgs/applications/graphics/smartdeblur/default.nix

```
@@ -1,24 +1,26 @@
-{ lib, stdenv, fetchFromGitHub, cmake, qt4, fftw }:
+{ lib, stdenv, fetchFromGitHub, cmake, fftw
+, qtbase, qmake, wrapQtAppsHook }:
 
 stdenv.mkDerivation rec {
   pname = "smartdeblur";
-  version = "unstable-2013-01-09";
+  version = "unstable-2018-10-29";
 
   src = fetchFromGitHub {
     owner = "Y-Vladimir";
     repo = "SmartDeblur";
-    rev = "9895036d26cbb823a9ade28cdcb26fd0ac37258e";
-    sha256 = "sha256-+EbqEpOG1fj2OKmlz8NRF/CGfT2OYGwY5/lwJHCHaMw=";
+    rev = "5af573c7048ac49ef68e638f3405d3a571b96a8b";
+    sha256 = "151vdd5ld0clw0vgp0fvp2gp2ybwpx9g43dad9fvbvwkg60izs87";
   };
 
-  preConfigure = ''
-    cd src
-  '';
+  sourceRoot = "source/src";
 
-  nativeBuildInputs = [ cmake ];
-  buildInputs = [ qt4 fftw ];
+  nativeBuildInputs = [ qmake wrapQtAppsHook ];
+  buildInputs = [ qtbase fftw ];
 
-  cmakeFlags = [ "-DUSE_SYSTEM_FFTW=ON" ];
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ./SmartDeblur $_
```

```suggestion
    cp ./SmartDeblur $out/bin/
```
or use install but I don't think that we should a special bash variable here.